app/forms/onboarding/request_form.rb

```diff
@@ -47,7 +47,7 @@ def proxy_auth_params
   end
 
   def test_api!
-    return unless model.deploy!
+    return unless ProxyDeploymentService.call(model, v1_compatible: true)
```

Comment:
I will leave it for now, but please put a comment that once APIcast V1 is gone we should remove it